Header header
uint8[360000] height
uint8[360000] density
uint8[360000] intensity
uint8[360000] ring
